docker-compose.yml

```diff
@@ -1,19 +1,23 @@
+version: '3'
 services:
-  app:
-    platform: linux/amd64
+  development:
     build:
-      context: ../../..
-      dockerfile: plugins/sqlfluff-templater-dbt/docker/Dockerfile.dev
+      context: .
+      dockerfile: ./docker/development/Dockerfile
+    environment:
+      - SSH_AUTH_SOCK=/ssh-agent
```

Comment:
What's this for?

Reply:
It's for using git inside the container. I always work in the shell generally, on a mac you need this env var to use ssh for git